### PR TITLE
Java installation instructions

### PR DESCRIPTION
**Pull request by saolsen**
Instructions on where to download the jar and how to put it on the classpath.

Is there any way we can dynamically inject the latest version? I didn't add maven instructions because they're pretty tedious. Will add those in when we're on maven central and it's easy to add to your project.

### CODE REVIEW

**Comment by saolsen** [comment]
Added the release thing we talked about in slack and made sure it's in sync with all other versions.

File: docs/getting-started/download/java.rst [hunk 1]
```diff
@@ -2,7 +2,13 @@
 Java
 ====
 
-.. todo:: Java installation
+The Java version of oso is available on Github. Go to the `Maven Repository <https://github.com/osohq/oso/packages/321403>`_ and download the latest jar.
+
+To use it, add it to the classpath for your project.
+
+    $ javac -classpath "oso-|release|.jar:." MyProject.java
```
Comment (samscott89) [inline docs/getting-started/download/java.rst:9]: Does this actually work? I'm surprised sphinx lets you do substitutions mid-text

Comment (saolsen) [reply docs/getting-started/download/java.rst:9]: Yep, 
Screen Shot 2020-07-25 at 6 14 18 PM

File: docs/spelling_allowed_words.txt [hunk 2]
```diff
@@ -58,5 +58,5 @@ iterables
 sqlite
 lookups
 classpath
-Github
+GitHub
```
Comment (samscott89) [inline docs/spelling_allowed_words.txt:61]:
```suggestion
```

(its on there already) - line 43

**Pull request merged**